dist/jquery.checkboxes-1.2.0.js

```diff
@@ -208,5 +200,9 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
 
     // Register data-api listeners.
     $(document).on('click.checkboxes.data-api', '[data-toggle^=checkboxes]', dataApiClickHandler);
-    $(document).on('ready.checkboxes.data-api', dataApiDomReadyHandler);
+    $(document).on('checkboxes.data-api.ready', dataApiDomReadyHandler);
+    $(function () {
+        $(document).trigger('checkboxes.data-api.ready');
+        $(document).off('checkboxes.data-api.ready');
+    });
```

Comment:
@jengjeng all these lines (203-207) can be reduced to a single:

```js
$(dataApiDomReadyHandler);
```

With that we will be ready to merge.